aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -239,9 +278,9 @@ impl<
 
     /// Remove all entries from transaction 'txn_idx' at access path 'key'.
     pub fn remove(&self, group_key: &K, txn_idx: TxnIndex, tags: HashSet<T>) {
```

Comment:
pass & to tags, and make sure executor does not clone but key and tags before calling the method.